src/test/java/org/fogbowcloud/manager/core/cloudconnector/LocalCloudConnectorTest.java

```diff
@@ -0,0 +1,359 @@
+		assertEquals(FAKE_ORDER_ID, this.localCloudConnector.getInstance(order).getId());
+	}
+	
+	// The order doesn't have an InstanceID, so the method an empty AttachmentInstance is returned with the same id of order.
+	@Test
+	public void testgetEmptyAttachmentInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(AttachmentOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.ATTACHMENT);
+	    when(this.order.getInstanceId()).thenReturn(null);
+	    when(this.order.getId()).thenReturn(FAKE_ORDER_ID);
+		assertEquals(FAKE_ORDER_ID, this.localCloudConnector.getInstance(order).getId());
+	}
+	
+	// The order doesn't have an InstanceID, so the method an empty ComputeInstance is returned with the same id of order.
+	@Test
+	public void testgetEmptyComputeInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(ComputeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
+	    when(this.order.getInstanceId()).thenReturn(null);
+	    when(this.order.getId()).thenReturn(FAKE_ORDER_ID);
+		assertEquals(FAKE_ORDER_ID, this.localCloudConnector.getInstance(order).getId());
+	}
+	
+	@Test
+	public void testDeleteInstanceWithoutInstanceID() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(ComputeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
+	    //when(computePlugin.requestInstance(any(ComputeOrder.class), any(Token.class))).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteComputeInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(ComputeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteVolumeInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(VolumeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.VOLUME);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteNetworkInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(NetworkOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.NETWORK);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteAttachmentInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(AttachmentOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.ATTACHMENT);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
```

Comment:
I did not understand the assert condition in this test. where is it?

Reply:
deleteInstance() method does nothing (just log). It delegates the task to the plugin. I call the method to make sure that any error happens. I can delete the test (decreasing the coverage)  or I can use a try/catch block with Assert.fail() into catch.

Reply:
@jvmafra I think you need to modify this method and also the testDeleteInstanceWithoutInstanceID, testDeleteComputeInstance, testDeleteVolumeInstance, testDeleteNetworkInstance, testDeleteAttachmentInstance to assert something. In this case, the assert is the forward of the call to the plugin. this is the expected behaviour. After that, I think the PR is ok

Reply:
Done